Brocfile.js

```diff
@@ -83,27 +83,6 @@ app.import('bower_components/font-awesome/fonts/FontAwesome.otf', { destDir: 'fo
 app.import('bower_components/cidcons/css/cidcons-codes.css');
 app.import('bower_components/cidcons/font/cidcons.woff', { destDir: 'fonts' });
 
-// Platform
```

Comment:
Totally forgot to do that part - yes we can remove them.

On Fri, Dec 4, 2015 at 3:26 PM Quinn Lee <notifications@github.com> wrote:

> In Brocfile.js
> <https://github.com/cid-harvard/atlas-subnational-frontend/pull/191#discussion_r46728314>
> :
>
> > @@ -83,27 +83,6 @@ app.import('bower_components/font-awesome/fonts/FontAwesome.otf', { destDir: 'fo
> >  app.import('bower_components/cidcons/css/cidcons-codes.css');
> >  app.import('bower_components/cidcons/font/cidcons.woff', { destDir: 'fonts' });
> >
> > -// Platform
>
> Can we remove these files from the vendor dir?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cid-harvard/atlas-subnational-frontend/pull/191/files#r46728314>
> .
>